src/api/wayfire/core.hpp

```diff
@@ -144,6 +144,8 @@ class compositor_core_t : public wf::object_base_t, public signal::provider_t
         wlr_xdg_foreign_registry *foreign_registry;
         wlr_xdg_foreign_v1 *foreign_v1;
         wlr_xdg_foreign_v2 *foreign_v2;
+
+        wlr_session_lock_manager_v1 *session_lock;
```

Comment:
Do we need to define this in core, since the functionality is implemented as a plugin?

Reply:
Left over from a previous attempt :-). Removed.